source/GameContainer.js

```diff
@@ -27,8 +26,22 @@ export default class GameContainer extends Pixi.Container {
     untilJunk(x, y) {
         if (this.countdownToJunk == 0) {
             this.countdownToJunk = Utility.randomNumber(Reference.JUNK_FREQUENCY_RANGE.lower, Reference.JUNK_FREQUENCY_RANGE.upper)
-            console.log(this.countdownToJunk)
+            // console.log(this.countdownToJunk)
             this.addChild(new Junk(x, y))
         } else this.countdownToJunk--
     }
-}
\ No newline at end of file
+
+    checkPlayerSpawn() {
+        for (var i = 0; i < Reference.controlTypeCount; i++) {
+            if (!Reference.ControlScheme.keys[i].inUse && (
+                Keyb.isDown(Reference.ControlScheme.keys[i].up) ||
+                Keyb.isDown(Reference.ControlScheme.keys[i].down) ||
+                Keyb.isDown(Reference.ControlScheme.keys[i].left) ||
+                Keyb.isDown(Reference.ControlScheme.keys[i].right))) {
```

Comment:
I notice we're only listening on only movement keys, not shooting, contrary to any key as listed in #41?